cask-cli.el

```diff
@@ -87,6 +87,33 @@
           (princ (s-pad-right (+ max cask-cli--table-padding) " " key))
           (princ (concat value "\n")))))))
 
+(defun cask-cli--missing-elpa-dependencies ()
+  "Return a list of direct dependencies that are missing."
+  (let ((bundle (cask-cli--bundle)))
+    (-remove
+     (lambda (dependency)
+       (let ((name (cask-dependency-name dependency)))
+         (or (epl-package-installed-p name)
+             (cask-linked-p bundle name))))
+     (cask-dependencies bundle))))
+
+(defun cask-cli--missing-dependencies-report (missing)
+  "Return a string report describing missing dependencies."
+  (->> missing
+       (-map #'cask-dependency-name)
+       (-map #'symbol-name)
+       (s-join ", ")))
```

Comment:
Please use a line break to separate the list of missing packages.  That's easier to read I think, and it's easier to feed into other programs like `grep` or whatever.